charts/aws-fluent-bit/values.yaml

```diff
@@ -1,18 +1,26 @@
 namespace:
-  create: true
+## Namespace creation is set to false as the namespace is expected to be created through the CLI since eksctl command may require.
+  create: false
   name: "calyptia-fluentbit"
 
+## The service account creation configuration is taken care by the eksctl command to contact the AWS marketplace API's to fetch the image.
 serviceAccount:
-  name: "eks-calyptia-fluentbit"
-  create: true
+  create: false
+## This is the name of the service account created by the eksctl create iamservice command.
+  name: "<Enter the name of the service account>"
   annotations:
-    eks.amazonaws.com/role-arn: "<Enter the ARN of the IAM service account>"
+## This is the Role ARN of the IAM role which should have the required policy and trust relationship setup using OIDC.
+    eks.amazonaws.com/role-arn: "<Enter the Role ARN of the IAM service account>"
```

Comment:
How do we get this? Can we automate it in any way?
Why do we need it as well as the service account, what happens if the annotation is not set?

Reply:
This would be the Role ARN that gets created through the eksctl create iamserviceaccount command
<img width="636" alt="image" src="https://user-images.githubusercontent.com/90174835/231419161-32905cd9-59dd-417c-b30e-2d25e8c02051.png">

**> Why do we need it as well as the service account, what happens if the annotation is not set?**
We don't really need this apart from the name of the service account which will be used in the pod.
I have included it so that there would be folks who prefer the service account as well is managed by the Helm.
Do you think we can leave it as is or shall I go ahead and remove this?

Reply:
If it is optional let us just leave it empty and only add if not empty - make a comment in the values file explaining it.